src/components/Imports/Providers/nextcloud/accountService.js

```diff
@@ -0,0 +1,31 @@
+export async function createNextcloudAccount(client, { login, password, url }) {
+  if (!login || !password || !url) {
+    throw new Error('Missing credentials for Nextcloud account creation')
+  }
+
+  const accountData = {
+    _type: 'io.cozy.accounts',
+    account_type: 'nextcloud',
+    auth: {
+      login,
+      password,
+      url
+    },
+    identifier: 'login',
+    name: login,
+    state: null
+  }
+
+  const res = await client.save(accountData)
+  // cozy-client usually returns the saved doc directly, but keep a safe fallback
+  return res && res.data ? res.data : res
+}
+
+export async function listNextcloudAccounts(client) {
```

Comment:
Not used? 

Also take a look at our naming convention https://github.com/cozy/cozy-guidelines?tab=readme-ov-file#naming-of-functions 

We should avoid the call to fetchJSON, client.query is our friend.